widgetastic_manageiq/expression_editor.py

```diff
@@ -181,6 +181,7 @@ def click_remove(self):
         self.browser.click(self.REMOVE)
 
     def click_commit(self):
+        self.COMMIT.wait_displayed()
```

Comment:
:ok_hand:

Reply:
Still seeing 5.10 failures, but this is a good addition

Reply:
there are also some other places where we don't wait enough. those failures have been hidden by try/catch before.

Reply:
testing by using `fill_strategy` now